contrib/images/slinky.sidecar.e2e.Dockerfile

```diff
@@ -16,7 +16,7 @@ EXPOSE 8080
 EXPOSE 8002
 
 COPY --from=builder /src/slinky/build/* /usr/local/bin/
-COPY --from=builder /src/slinky/config/local /etc/slinky/default_config
```

Comment:
i feel like this should be a `dydx` specifc image.  We should still maintain a default one